SPECS/complianceascode/complianceascode.spec

```diff
@@ -0,0 +1,44 @@
+%global debug_package   %{nil}
+
+Summary:        package for creating security policy content for various platforms
+Name:           complianceascode
+Version:        1.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Base
+URL:            https://aka.ms/mariner
```

Comment:
Can't use this URL since it links to an internal site. Let's point to our own github for now.